examples/graph/gqa_training.cpp

```diff
@@ -0,0 +1,567 @@
+/*******************************************************************************
+* Copyright 2024-2025 Intel Corporation
```

Comment:
```suggestion
* Copyright 2025 Intel Corporation
```

Reply:
And add this file to https://github.com/uxlfoundation/oneDNN/blob/main/examples/CMakeLists.txt#L94 .